README.md

```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
 ## goes/
 
 Utilties for using NOAA's GOES-16/17 satellite data, which is publicly accessible in Amazon S3.
+
+## NAS
+
+This project is meant to be deployed both locally and on NASA's NEX Super Computers.
+1. For detailed documentation on a number of topics and bugs at NAS please see [here.](https://www.nas.nasa.gov/hecc/support/kb/)
+1. For an overview of many common NAS tasks see [here.](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html)
+1. For an overview of NAS Compute Architecture see [here.](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html)
+1. Support can be reached easily by phone at 800-331-8737 and <support@nas.nasa.gov>.
+
+### Storage
+
+1. All data should be stored on Lou. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/the-lou-mass-storage-system_371.html).
+1. For all data storage purposes, please store your data on ` Lou:/u/your_nas_username`
+1. `/nobackup/your_nas_username` is cleared regularly regularly. `home4/your_nas_username` is not.
+1. NAS has much of the GOES16 data we are working with available in `nex/datapool/goes16`, accessible from pfe.
```

Comment:
Maybe expanding what PFE means, or a quick overview of the different compute servers?